docs/reference/analysis/tokenizers/simplepatternsplit-tokenizer.asciidoc

```diff
@@ -0,0 +1,107 @@
+[[analysis-simplepatternsplit-tokenizer]]
+=== Simple Pattern Split Tokenizer
+
+experimental[]
+
+The `simplepatternsplit` tokenizer uses a regular expression to split the
+input into terms at pattern matches. The set of regular expression features it
+supports is more limited than the <<analysis-pattern-tokenizer,`pattern`>>
+tokenizer, but the tokenization is generally faster.
+
+This tokenizer does not produce terms from the matches themselves. To produce
+terms from matches using patterns in the same restricted regular expression
+subset, see the <<analysis-simplepattern-tokenizer,`simplepattern`>>
+tokenizer.
+
+This tokenizer uses http://lucene.apache.org/core//6_5_1/core/org/apache/lucene/util/automaton/RegExp.html[Lucene regular expressions].
```

Comment:
I wonder about linking to a specific version of the docs. Are these going to rapidly go stale? Can we link based on the Lucene version constant (`lucene_version_path`)?